public/emails/ng_alert_notification.html

```diff
@@ -210,6 +210,21 @@
 {{Subject .Subject "{{.Title}}"}}
 
 {{ define "alert" }}
+
+  {{if ne .ImageURL "" }}
```

Comment:
This adds a row just after the subject, before the value string. See included image in this comment.
<img width="1104" alt="Screenshot 2022-06-08 at 12 21 38 PM" src="https://user-images.githubusercontent.com/1951900/172530970-124fbe18-d1cc-4d04-b02f-831579c989d9.png">